src/core/server/base.js

```diff
@@ -13,19 +13,21 @@ import { match } from 'react-router';
 import { ReduxAsyncConnect, loadOnServer } from 'redux-async-connect';
 
 import WebpackIsomorphicTools from 'webpack-isomorphic-tools';
-import WebpackIsomorphicToolsConfig from 'config/webpack-isomorphic-tools';
+import WebpackIsomorphicToolsConfig from 'webpack-isomorphic-tools-config';
 
 import config from 'config';
 import { setJWT } from 'core/actions';
 
+const env = config.util.getEnv('NODE_ENV');
+const isDeployed = ['stage', 'dev', 'production'].indexOf(env) > -1;
```

Comment:
It only seems to be used for SRI which we can do any time the server is running with built files, not just when it is deployed somewhere (although these roughly equivalent).

I guess if you just call `npm start:search` locally the `NODE_ENV` won't be development, however I don't think anything would stop you from doing so.